### DISCUSSION TITLE
Wildcard regex in velero command line

### DISCUSSION THREAD

**Discussion by manish-jangra** [vmware-tanzu/velero#6941] [General discussions]
is there any way to use wildcard or regex in velero command line like below --

`velero backup create test --include-namespaces 'test-*'`

**Comment by sseago** [comment]
This is not currently supported. There is an open issue for this (https://github.com/vmware-tanzu/velero/issues/1874 ) but implementation of this feature has not started at this time.

**Reply by manish-jangra** [reply]
It works for --exclude-namespaces but not for --include-namespaces. Any idea, when this can be available? or Can I pick this open item and work through.

**Reply by sseago** [reply]
Nobody is currently working on it, so if you want to submit a PR that implements this, we'd be happy to review it.
